src/constant.js

```diff
@@ -0,0 +1,23 @@
+export const MAX_CAR_COUNT = 50;
+export const MAX_TRY_COUNT = 500;
+
+export const CONSOLE_MESSAGE = Object.freeze({
```

Comment:
```suggestion
export const CONSOLE_MESSAGES = Object.freeze({
```
`CONSOLE_MESSAGES`와 같이 복수형태를 드러내도 좋을 것 같네요!